### ISSUE TITLE
Typescript declaration is wrong

### ISSUE THREAD

**Issue by DanilAndreev** [frinyvonnick/node-html-to-image#127]
Hello, assigns ```module.exports``` a function, but declared in d.ts file that function exported from module as default value. 
In ES6 modules default export has ```module.exports.default``` route. That causes errors when trying to use it as shown in readme.

```
import * as nodeHtmlToImage from 'node-html-to-image'

// const nodeHtmlToImage = require('node-html-to-image');

// @ts-ignore
nodeHtmlToImage({
    output: './image.png',
    html: `<html>
    <head>
      <style>
        body {
          width: 2480px;
          height: 3508px;
        }
      </style>
    </head>
    <body>Hello world!</body>
  </html>
  `
}).then(() => console.log('The image was created successfully!'))
```

**Comment by frinyvonnick** [comment]
Hello @DanilAndreev 👋 

Thank you for opening this issue 🙏  Would you be interested in opening a pull request to fix it?

**Comment by DanilAndreev** [comment]
Ok

**Comment by DanilAndreev** [comment]
If you want - I can fully translate your code to typescript.

**Comment by frinyvonnick** [comment]
Published in [v3.2.1](https://github.com/frinyvonnick/node-html-to-image/releases/tag/v3.2.1) 🚀
